web-animations-2/Overview.bs

```diff
@@ -969,6 +969,34 @@ Add the following sentence:
 
 </div>
 
+### Calculating the progress of an animation ### {#the-progress-of-an-animation}
+
+  An animation's <dfn export for=animation>progress</dfn> is the ratio of its
+   [=animation/current time=] to its [=associated effect end=].
+
+  <div algorithm="calculate animation progress">
+    The [=animation/progress=] of an [=animation=], |animation|, is calculated as follows:
+
+    <dl class=switch>
+
+    : If <em>any</em> of the following are true:
+      * |animation| does not have an [=animation/associated effect=], or
+      * |animation|'s [=associated effect end=] is zero, or
+      * |animation|'s [=animation/current time=] is an [=unresolved=] time value,
+    ::
+        |animation|'s [=animation/progress=] is null.
+    : If |animation|'s [=associated effect end=] is infinite,
+        |animation|'s [=animation/progress=] is zero.
```

Comment:
Can the author tell if we're active (after the start time) or inactive (before the start time)?